index.html

```diff
@@ -1,77 +1,105 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- displays site properly based on user's device -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- displays site properly based on user's device -->
 
   <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
-  
+  <link rel="stylesheet" href="style.css">
+
   <title>Frontend Mentor | Manage landing page</title>
 
   <!-- Feel free to remove these styles or customise in your own stylesheet 👍 -->
   <style>
-    .attribution { font-size: 11px; text-align: center; }
-    .attribution a { color: hsl(228, 45%, 44%); }
+    .attribution {
+      font-size: 11px;
+      text-align: center;
+    }
+
+    .attribution a {
+      color: hsl(228, 45%, 44%);
+    }
   </style>
 </head>
-<body>
-
-  Pricing
-  Product
-  About Us
-  Careers
-  Community
 
-  Get Started
+<body>
+  <div id="header">
+    
+    <nav>
+      <ul>
+        
+
+       <li>
+        <img src="/images/logo.svg" />
+      </li> 
+        
+
+        <li>Pricing </li>
+        <li>Product </li>
+        <li>About Us </li>
+        <li>Careers </li>
+        <li>Community </li>
+
+      <button disabled="disabled" class="btn-one">Get Started</button>
+      </ul>
+    </nav>
+  </div>
+<section>
+<div class="box-one">
 
-  Bring everyone together to build better products.
+ <h1 class="main">Bring everyone together to build better products. </h1> 
 
-  Manage makes it simple for software teams to plan day-to-day 
+  Manage makes it simple for software teams to plan day-to-day
```

Comment:
It's advised to always put your texts in any tag other than div. Only leave text in div if there is no other option. In this case, this text can be placed in a `<p>` tag